external/wakaama/examples/client/connectivity_interface.c

```diff
@@ -45,8 +45,8 @@ void get_interface_name(char *mac)
     strcpy(mac,"eth0");
 #elif defined(CONFIG_NET_802154)
     strcpy(mac,"wlan0");
-#elif defined(CONFIG_ARCH_BOARD_SIDK_S5JT200)
-    strcpy(mac, "wl1");
+#elif defined(CONFIG_WIFI_MANAGER)
+    strcpy(mac,CONFIG_WIFIMGR_STA_IFNAME);
 #endif
```

Comment:
if CONFIT_WIFI_MANAGER is not defined, we can't use wireless lan interface. do we need to define default setting?

Reply:
@sangwon03 Change completed.